.github/DISCUSSION_TEMPLATE/transfer.yml

```diff
@@ -0,0 +1,32 @@
+title: "Transfer "
+labels: [question]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        It's not possible to transfer a parser to an organization that you're not an admin of,
+        and we will not hand admin rights to everyone. Instead, you must temporarily transfer the
+        repository to one of the [maintainers](https://github.com/orgs/tree-sitter-grammars/people).
```

Comment:
would be nice for every org member to set their visibility to public to make this list more accurate/useful

Reply:
(Or set it to private if they don't want to handle transfer requests)